cratetorrent/src/metainfo.rs

```diff
@@ -81,7 +81,8 @@ pub struct Metainfo {
     /// The trackers that we can announce to.
     /// The tier information is not currently present in this field as
     /// cratetorrent doesn't use it. In the future it may be added.
-    pub trackers: Vec<Url>,
+    /// The boolean is whether said tracker uses UDP or not
+    pub trackers: Vec<(Url, bool)>,
```

Comment:
I think it's better to define something like
```rust
struct TrackerUrl {
    url: Url,
    is_udp: bool
}
```
and use `Vec<TrackerUrl>` instead of `Vec<(Url, bool)>`

Reply:
I changed it